.github/workflows/pytest.yml

```diff
@@ -17,17 +17,12 @@ jobs:
       uses: actions/setup-python@master
       with:
         python-version: ${{ matrix.python-version }}
-    - name: apt-get
```

Comment:
Should we just delete this file?

Reply:
dunno maybe we add tests? I'll remove it I guess